### ISSUE TITLE
Allow suppressing the Producer metadata

### ISSUE THREAD

**Issue by ahumeau** [Kozea/WeasyPrint#2118]
Hi there,

First of all, thanks for the amazing project.

I am trying to find a way to remove the `Producer` metadata written by Weasyprint in order not to leak info about what software and version I am using.

I tried calling `write_pdf` with `custom_metadata=True` and setting `<meta name="Producer" content="">` in my document but that does not work because [all meta tag names are lowercased when extracting custom metadata from HTML](https://github.com/Kozea/WeasyPrint/blob/78d864bdc40716b43d47b778177de43da16ddb8d/weasyprint/html.py#L255) which means that instead of overriding the `Producer` metadata, I just add a `producer` metadata.

Best

**Comment by grewn0uille** [comment]
Hi @ahumeau,

Overwriting Producer metadata has already been discused in this PR: https://github.com/Kozea/WeasyPrint/pull/1611.

TL;DR, we don’t want to allow that, but if you really want to do it, you can use a [finisher function](https://doc.courtbouillon.org/weasyprint/stable/api_reference.html#weasyprint.HTML.write_pdf)!

**Comment by ahumeau** [comment]
Hi @grewn0uille,

I am sorry I did not find that previous issue and thanks for the workaround!

And again, thanks for your time and work.

**Comment by TheOnlyWayUp** [comment]
For others who stumbled past this issue, here's a working example
```py
  def finisher(self, document: weasyprint.Document, pdf: pydyf.PDF):
      pdf.info["completed"] = pydyf.String("true")  # Custom
      pdf.info["title"] = pydyf.String("Hi!")  # Standard
```

Any field can be written to, including custom ones like in the example.
